recourse_methods/mrmc_method.py

```diff
@@ -145,15 +145,15 @@ def __init__(self,
         
         Args:
             k_directions: The number of clusters (and recourse directions) to generate.
-            preprocessor: The dataset preprocessor.
+            adapter: The dataset adapter.
```

Comment:
Same with the other nit. This helps clarify the what this does and reduces the need to jump between different files to understand what this object is / how it's used.